astarte/device/introspection.py

```diff
@@ -83,9 +84,9 @@ def get_all_interfaces(self):
         list
             The list of all Interfaces in the Introspection
         """
-        return self.__interfaces_list.values()
+        return list(self.__interfaces_list.values())
```

Comment:
```suggestion
        return [self.__interfaces_list.values()]
```

Reply:
`list()` here is not used to **make** a list, but to make clear that the output value is of the `list` type. 
The function `dict.values()` outputs a `_dict_values` that is a disguised list, so no square brackets are needed.